src/app/core/services/partner.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore } from '@angular/fire/firestore'
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PartnerService {
```

Comment:
This service should extend the abstract class DataService (shared/models/data-service)